packages/opentelemetry-context-async-hooks/src/AbstractAsyncHooksContextManager.ts

```diff
@@ -91,12 +89,12 @@ export abstract class AbstractAsyncHooksContextManager
    * @param context the context we want to bind
    */
   private _bindEventEmitter<T extends EventEmitter>(
-    target: T,
+    ee: T,
```

Comment:
please update doc too and I think there is also a typo in doc `a instance` should be `an instance` (if this remains)